.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,7 +19,7 @@ Learn more about contributing: [CONTRIBUTING.md](https://github.com/nf-core/rnas
     - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnaseq/tree/master/.github/CONTRIBUTING.md)
     - [ ] If necessary, also make a PR on the nf-core/rnaseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
 - [ ] Make sure your code lints (`nf-core lint`).
-- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
+- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
```

Comment:
I guess this has been discussed already, but why not set the parameter in the `test.config`? I don't see the point of providing it several times in the `ci.yml` file, plus as user, the command is simpler to run tests

Reply:
After, reviewing the whole code, I see that the reason is that `outdir` param is removed from the `nextflow.config` but still I think it should be set by default in the `test.config`.

Reply:
If we set it to `./results` in the `test.config` then we end up in the same position where we end up using relative paths when `-profile test` is used in the Cloud? The main problem is that there isn't a sensible default that works yet.

Reply:
I see the cloud part is what I missed, fair enough